examples/object_detection_2d/object_detection_2d/constants.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict
+
+DATASET = "coco-2014-val"
+WORKFLOW = "Object Detection"
+
+S3_IMAGE_LOCATION = "s3://kolena-public-datasets/coco-2014-val/imgs/"
+S3_BUCKET = "kolena-public-datasets"
+S3_ANNOTATION_FILE_PATH = "coco-2014-val/meta/instances_val2014.json"
```

Comment:
These constants seem to be inconsistent with the usage of `S3_BUCKET`.  Can we either drop `s3://kolena-public-datasets` from `S3_IMAGE_LOCATION` or add `s3://kolena-public-datasets` to `S3_ANNOTATION_FILE_PATH`?